src/Shared/Program.cs

```diff
@@ -10,11 +11,17 @@ public class Program
     {
         public static void Main(string[] args)
         {
+            var assembly = Assembly.GetEntryAssembly();
+
+            string regexMatch = Regex.Match(assembly.FullName.Split(',')[0], "\\d{1,2}").Value;
```

Comment:
Is there a reason to use `FullName` and split on the comma rather than just use `Name`?